src/WikiParsing/Articles/VerbArticle.fs

```diff
@@ -45,7 +47,27 @@ let getImperatives verb =
     let data = getVerbProvider verb
     let answer = data.Tables.``Časování[editovat]2``.Rows.[0].``Číslo jednotné - 2.``
     getForms answer
+
+let removeHovorově (w:string) =
+    let parenthesis = w.IndexOf("(")
+    if parenthesis >= 0 then
+        w.Substring(0, parenthesis - 1)
+    else w
```

Comment:
In theory, this should not be needed - take a look into the `getForms` implementation in the `WikiString`. If you have discovered some different way of writing that label, better extend/adjust and test the logic there.